### ISSUE TITLE
Страница списка проектов

### ISSUE THREAD

**Issue by simlex-titul2005** [simlex-titul2005/goldfish.pro#11]
Обсуждаем http://goldfish.pro/siteprojects

**Comment by EugeneKormin** [comment]
Нужно добавить основные параметры сайта, цены, какие задачи ставились заказчиком, как они выполняются (с разрешения владельца)

**Comment by simlex-titul2005** [comment]
Это на детализированной странице проекта

**Comment by EugeneKormin** [comment]
ну так и на этой можно самые общие,хотя бы цену, задачу, процент выполнения

**Comment by simlex-titul2005** [comment]
да, можно , только необходим первоначальный список параметров для проекта

**Comment by EugeneKormin** [comment]
По идее - это внутренняя статистика сайта,надо отдельно договариваться с владельцем сайта, возможно за скидку.

**Comment by simlex-titul2005** [comment]
лучше просто вывести публичную статистику

**Comment by EugeneKormin** [comment]
можно так, но проблема в том,что любая статистика -это собственность сайта заказчика, нужно у владельца сайта спросить разрешения

**Comment by simlex-titul2005** [comment]
Тогда нужно определить, какие параметры являются приватными и на их отображение на автомате справшивать разрешение по email заказчика и в случае его одобрения - показывать их

**Comment by EugeneKormin** [comment]
по идее все являются...(((

**Comment by EugeneKormin** [comment]
скорее сказать, какие не являются...

**Comment by simlex-titul2005** [comment]
публичные параметры:
1. название проекта
2. дата начала разработки
3. исполнители
4. предварительная дата завершения
5. этапы : что нужно сделать, но сроки не указывать

**Comment by EugeneKormin** [comment]
1. этапы : что нужно сделать
   Вот насчет этого не уверен...

**Comment by simlex-titul2005** [comment]
тогда пока оставим 4
но отсюда вылезает уже что-то
1. Необходимо к каждому проекту прикреплять исполнителей

**Comment by EugeneKormin** [comment]
а вот исполнители - это скорее открытая информация
